packages/main/src/Popup.js

```diff
@@ -535,7 +549,7 @@ class Popup extends UI5Element {
 	 * @protected
 	 */
 	get _ariaLabel() {
-		return this.accessibleName || undefined;
+		return getEffectiveAriaLabelText(this) || undefined;
```

Comment:
When using ```getEffectiveAriaLabelText``` you don't need the fallback value of undefined. This case is handled.

Reply:
Done